config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   namespace :api do
     post 'login', to: 'users#login'
+    get 'golfer/:id/name', to: 'users#name'
```

Comment:
You should create a more general endpoint that returns the whole user resource.

Check out https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions for REST conventions. It should be `golfers/:id`